candi/cloud-providers/aws/layouts/standard/base-infrastructure/main.tf

```diff
@@ -180,10 +180,75 @@ resource "aws_iam_instance_profile" "node" {
 }
 
 resource "aws_key_pair" "ssh" {
-  key_name = local.prefix
+  key_name   = local.prefix
   public_key = var.providerClusterConfiguration.sshPublicKey
 
   tags = merge(local.tags, {
     Cluster = local.prefix
   })
 }
+
+// vpc peering
+
+locals {
+  peer_vpc_ids = lookup(var.providerClusterConfiguration, "peeredVPCs", [])
```

Comment:
It seems that appended terraform code is the same for both layouts. Can't we just move it to module to reduce copy-paste?

Reply:
Yes we can. Made vpc-peering module.